site/content/entry/yuzu-progress-report-sep-2022/index.md

```diff
@@ -0,0 +1,293 @@
+
+The old SD card icon has been {{< gh-hovercard "8902" "replaced with a new colourful microSD Card icon" >}} by [Dev-draco](https://github.com/Dev-draco).
+
+{{< imgs
+	"./sdicon.png| Small details matter"
+  >}}
+
+[Tachi107](https://github.com/Tachi107) {{< gh-hovercard "8945" "fixed a few minor typos" >}} within the yuzu source code. 
+
+german77 modified the input profiles He also fixed the {{< gh-hovercard "8948" "sorting to now sort them by name." >}}
+
+While the colourful theme has been the default for a while, some icons remained in a "Default" theme directory.
+But as grabbing icons from other themes isn't supported on linux, this was broken.
+Docteh fixed this by {{< gh-hovercard "8906" "moving all the icons to the colourful theme." >}}
+
+Since the **Debug** configuration tab was getting crowded, Docteh {{< gh-hovercard "8854" "made it scrollable." >}}
+
+{{< single-title-imgs
+    "While this change helps Windows, Linux users will benefit the most of it"
+    "./scrollbug.png"
+    "./scrollfix.png"
+    >}}
+
+Docteh also fixed the {{< gh-hovercard "8909" "broken help page hyperlink on the `TAS` configuration window" >}}.
+
+{{< imgs
+	"./tas.png| Poeple seem to forget that blue underlined text is an hyperlink intended to be clicked"
+  >}}
+
+## Miscellaneous 
+
+For a long time, any yuzu bug reports from Windows users had to be manually reproduced and investigated by the team.
+Linux users had the option of providing stack traces, but there wasn't an easy way for Windows users to provide similar debug info.
+
+To overcome this, toastUnlimited implemented {{< gh-hovercard "8682" "support to create Windows crash dumps within yuzu" >}} itself.
+This allows any layman on Windows to easily obtain a crash dump without jumping through various developer intended hoops.
+But since this feature has a big performance impact, this was added behind an option in the **Debug** settings for users to access.
+You can find it in `Emulation > Configure… > General > Debug > Create Minidump After Crash`.
+
+{{< imgs
+	"./minidump.png| Only use it if you want to debug a game or to pass the file to a developer, the performance loss is big!"
+  >}}
+
+Since the dumps use modern Windows SDK features, there’s a risk that this change can break compatibility with older Windows options.
+yuzu only officially supports Windows versions 10 1803 and newer. 
+And Linux, of course. It's faster than Windows.
+
+## Hardware section
+
+Users started reporting crashes when booting up `Xenoblade Chronicles 3` on AMD Radeon GPUs running official AMD drivers (either the Windows driver, or the Linux amdvlk driver).
+We’re investigating and already passed AMD the relevant information.
+
+Affected users can try running [Mainline 1188](https://github.com/yuzu-emu/yuzu-mainline/releases/tag/mainline-0-1188) (doesn’t seem to be a consistent fix), or just run the game using OpenGL.
+The Mesa RADV driver is unaffected.
+
```

Comment:
Maybe a note saying  {{< gh-hovercard "8849" "astc: Enable parallel CPU astc decoding " >}} is merged now as this happened 3 days after the previous report was published and Maxwell and Pascal GPUs can upgrade their drivers, or if not, what the current status is for those GPUs and Vulkan.

Reply:
Thanks, added.